src/Makefile.am

```diff
@@ -1 +1,3 @@
-dist_bin_SCRIPTS = eos-google-chrome-app
+dist_bin_SCRIPTS =		\
+	eos-google-chrome-app	\
+	eos-google-chrome-dummy-cmd
```

Comment:
Hmm... I don't really like this, adding a new script just to print a message when run from inside the flatpak seems a bit overkill to me, specially considering that you can detect from inside the eos-google-chrome-app when you're being run from inside a flatpak sandboxed environment.

What I would do instead would be this:
  1. In the JSON file, in eos-build, change the `command` key to `eos-google-chrome-app`
  2. Inside eos-google-chrome-app, do an early check for `$XDG_RUNTIME_DIR/flatpak-info`: if it's reachable it means you're being run inside a flatpak environment -> Print the message discussed below and exit. If it's not reachable, it means this script is being run from outside (i.e. from eos-google-chrome-helper), in which case you proceed normally.